config/locales/en.yml

```diff
@@ -381,6 +381,7 @@ en:
           audio: Audio - upload multiple audio files
           uncut: Audio - upload a single file and then segment
           video: Video
+          override: Override - specify external media url
```

Comment:
Recommend alphabetizing this list.

Reply:
Just noting... sorting these in en.yml won't sort them in the UI.  Since it's just pulling the order of the rails enum.  We'd have add a sort-by-transated-value to `def episode_medium_options`.

Reply:
looks like they are sorted based on the order in the `mediums` enum in the `episode_media.rb`
https://github.com/PRX/feeder.prx.org/blob/9dc036c81f162ae2f579a1e9f50d5f97b1c9cd0b/app/helpers/episodes_helper.rb#L116-L118

and that sort order is:
https://github.com/PRX/feeder.prx.org/pull/1163/files#diff-bda765f89a866286ad1f1b6a4cf2ed1eed4bb2b6859d7e354da995bfaf139f32R7

Reply:
example order:
<img width="392" alt="image" src="https://github.com/user-attachments/assets/69ca8257-cc24-4142-b2a3-256b149efd13" />